hs_app_netCDF/receivers.py

```diff
@@ -14,15 +14,17 @@
 from hs_core.hydroshare import utils
 from hs_app_netCDF.forms import VariableValidationForm, OriginalCoverageForm, VariableForm
 from hs_app_netCDF.models import NetcdfResource
-import nc_functions.nc_utils as nc_utils
-import nc_functions.nc_dump as nc_dump
-import nc_functions.nc_meta as nc_meta
+import hs_file_types.nc_functions.nc_utils as nc_utils
+import hs_file_types.nc_functions.nc_dump as nc_dump
+import hs_file_types.nc_functions.nc_meta as nc_meta
 
 
 # receiver used to extract metadata after user click on "create resource"
 @receiver(pre_create_resource, sender=NetcdfResource)
 def netcdf_pre_create_resource(sender, **kwargs):
-
+    # Pabitra: Though there is some common code between netcdf file type and here as it relates
```

Comment:
@pkdash Agreed it'd be better to move metadata extraction into common code to have single point of truth for better maintainability so we don't have to make changes in multiple places in the future.